infra/feast-operator/internal/controller/services/services.go

```diff
@@ -216,6 +235,17 @@ func (feast *FeastServices) removeFeastServiceByType(feastType FeastServiceType)
 	return nil
 }
 
+func (feast *FeastServices) removeRoute(feastType FeastServiceType) error {
+	if !isOpenShift {
+		return nil
+	}
+	route := feast.initRoute(feastType)
+	if err := feast.Handler.DeleteOwnedFeastObj(route); err != nil && !apierrors.IsNotFound(err) {
+		return err
+	}
```

Comment:
```suggestion
	if err := feast.Handler.DeleteOwnedFeastObj(route); err != nil {
		return err
	}
```
`!apierrors.IsNotFound(err)` is already handled by the `DeleteOwnedFeastObj` method

Reply:
change done